htp/htp_response_generic.c

```diff
@@ -256,6 +256,14 @@ htp_status_t htp_process_response_header_generic(htp_connp_t *connp, unsigned ch
     htp_header_t *h_existing = htp_table_get(connp->out_tx->response_headers, h->name);
     if (h_existing != NULL) {
         // Keep track of repeated same-name headers.
+        if (h_existing->flags & HTP_FIELD_REPEATED) {
+            // This is at least the third occurence for this header.
+            htp_log(connp, HTP_LOG_MARK, HTP_LOG_WARNING, 0, "Repetition for header");
```

Comment:
if we only call this if is not set HTP_FIELD_REPEATED, the we'd just call it once right?
```c
if (h_existing != NULL) {
    if (h_existing->flags & HTP_FIELD_REPEATED == 0) {
        htp_log(connp, HTP_LOG_MARK, HTP_LOG_WARNING, 0, "Repetition for header");
    } else {
        bstr_free(h->name);
        bstr_free(h->value);
        free(h);
        return HTP_OK;
    }
}

Reply:
Yes, but this will issue a warning starting from the first repetition. Isn't it too much ?

Reply:
I think warning on any duplicate header is already useful. I don't think it should really happen in normal traffic. Or do you think I'm missing something?

Reply:
I saw it happen on "normal traffic", some duplicate secure header like `X-XSS-Protection:1` to be really sure it is secure :-)
So yes, this was misconfiguration, and it was worth reporting...